shared/networking/2.0.md

```diff
@@ -0,0 +1,202 @@
+# Networking on resinOS 2.0
+
+* [Introduction](#introduction)
+* [Wifi Setup](#wifi-setup)
+* [Setting a Static IP](#setting-a-static-ip)
+* [Cellular Modem Setup](#cellular-modem-setup)
+* [Network Requirements](#network-requirements)
+
+## Introduction
+
+In resinOS 2.0 the connectivity management has been change from [connman][connman-link] to [NetworkManager][networkmanager-link]. NetworkManager give the OS a bit more flexibility in the network setup and in conjunction with [ModemManager][modemmanager-link] allows resinOS to offer first class [GSM/Cellular support](#setup-a-cellular-connection).   
```

Comment:
NetworkManager give the OS a bit more flexibility in the network setup -> NetworkManager allows resinOS more flexibility when configuring the network. 

(or just drop the `a bit`)